monitor_check.py

```diff
@@ -13,17 +13,17 @@
 from collections import Counter
 
 
-MONITOR = 'https://copr.fedorainfracloud.org/coprs/g/python/python3.9/monitor/'
-INDEX = 'https://copr-be.cloud.fedoraproject.org/results/@python/python3.9/fedora-rawhide-x86_64/{build:08d}-{package}/'  # keep the slash
+MONITOR = 'https://copr.fedorainfracloud.org/coprs/g/python/python3.10/monitor/'
+INDEX = 'https://copr-be.cloud.fedoraproject.org/results/@python/python3.10/fedora-rawhide-x86_64/{build:08d}-{package}/'  # keep the slash
 PDC = 'https://pdc.fedoraproject.org/rest_api/v1/component-branches/?name=master&global_component={package}'
-PACKAGE = re.compile(r'<a href="/coprs/g/python/python3.9/package/([^/]+)/">')
-BUILD = re.compile(r'<a href="/coprs/g/python/python3.9/build/([^/]+)/">')
+PACKAGE = re.compile(r'<a href="/coprs/g/python/python3.10/package/([^/]+)/">')
+BUILD = re.compile(r'<a href="/coprs/g/python/python3.10/build/([^/]+)/">')
 RESULT = re.compile(r'<span class="build-([^"]+)"')
 RPM_FILE = "<td class='t'>RPM File</td>"
-TAG = 'f33'
+TAG = 'f35'
```

Comment:
This needs to be rawhide tag. I.e. now 34, only later 35. it is used to determine if a package is retired on rawhide.

Reply:
I have updated PR, do we need to change something in already opened bugzillas?

Reply:
Thanks. Not really, this constant is used when auto-triaging failures, not in reporting.